### DISCUSSION TITLE
复现了一个曾经被报告的bug

### DISCUSSION THREAD

**Discussion by mcenahle** [Meloong-Git/PCL#3190] [提问] [answered]
<img width="790" alt="image" src="https://github.com/Hex-Dragon/PCL2/assets/85427807/2653cae2-e7a7-4a71-ba57-1e003f88203c">
今天打开PCL2玩，不知道怎么就出现了这个问题。

**Comment by allMagicNB** [comment, marked as answer]
#2982

**Comment by allMagicNB** [comment]
~~不是，好歹给我点个Mark as answer~~
